### PR TITLE
update module api reference

### PR DESCRIPTION
**Pull request by oranagra**

### CODE REVIEW

**Comment by oranagra** [comment]
@zuiderkwast @itamarhaber i assume this is mainly https://github.com/redis/redis/pull/8442
please take a look at merge.

**Comment by itamarhaber** [comment]
Pulled and checked - looks great. Merging and here's to 6.2.2!

File: topics/modules-api-ref.md [hunk 1]
```diff
@@ -1732,22 +2152,40 @@ Example code fragment:
        // Do something with myval.
      }
 
-This API is documented here: [https://redis.io/topics/modules-intro](https://redis.io/topics/modules-intro)
+This API is documented here:[ https://redis.io/topics/modules-intro]( https://redis.io/topics/modules-intro)
```
Comment (zuiderkwast) [inline topics/modules-api-ref.md:2155]: These links look wrong. It's caused by a change in the ruby script here: https://github.com/redis/redis/pull/8442/files#r615932389

Comment (zuiderkwast) [reply topics/modules-api-ref.md:2155]: The links work anyway, so no disaster. The generated HTML code for this link is `<p>This API is documented here:<a href="https://redis.io/topics/modules-intro"> https://redis.io/topics/modules-intro</a></p>`.

Comment (zuiderkwast) [reply topics/modules-api-ref.md:2155]: Fixed here: https://github.com/redis/redis/pull/8818

**Pull request merged**